controllers/controllers/services/instances/upsi/controller_test.go

```diff
@@ -46,6 +46,13 @@ var _ = Describe("CFServiceInstance", func() {
 				},
 			}
 			Expect(adminClient.Create(ctx, instance)).To(Succeed())
+
+			Expect(k8s.Patch(ctx, adminClient, instance, func() {
```

Comment:
No need to set last operation here - the controller will do this in the reconciliation loop